inferschema.go

```diff
@@ -0,0 +1,33 @@
+package airbyte
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/bitstrapped/airbyte/schema"
+	"reflect"
+)
+
+// Infer schema translates golang structs to JSONSchema format
+func InferSchemaFromStruct(i interface{}, logTracker LogTracker) Properties {
+	var prop Properties
+
+	s, err := schema.Generate(reflect.TypeOf(i))
+	if err != nil {
+		logTracker.Log(LogLevelError, fmt.Sprintf("Generate schema error: %d", err))
```

Comment:
Standard go practice also suggests errors should start with lowercase. I wonder if we should follow that practice when logging to airbyte as well.

Reply:
Also good to know, done!